ConnectorAutocadCivil/ConnectorAutocadCivil/Utils.cs

```diff
@@ -354,9 +458,13 @@ public static void SetPropertySets(this Entity entity, Document doc, List<Dictio
     {
       propertySets = PropertyDataServices.GetPropertySets(obj);
     }
-    catch (Exception e)
-    { }
-    if (propertySets == null)
+    catch (Autodesk.AutoCAD.Runtime.Exception e) when (!e.IsFatal())
```

Comment:
I don't think we need `IsFatal` here as an `Autocad` specific exception doesn't overlap with any of the exceptions we consider "unrecoverable" in the IsFatal implementation (they all derive from System.Exception)